configs/1vcs_1mld+2sld.yaml

```diff
@@ -0,0 +1,25 @@
+port_configs:
+  - type: USP
+  - type: DSP
+  - type: DSP
+  - type: DSP
+virtual_switch_configs:
+  - upstream_port_index: 0
+    vppb_counts: 4
+    initial_bounds: [1,1,2,3]
+multi_devices:
+  - mld_port_index: 1
```

Comment:
Don't need `multi_devices` section since the presence of `mld_port_index` should be a sufficient evidence of MLD.